src/pages/Aggregator/AggregatorInfo/AggregatorInfo.component.tsx

```diff
@@ -32,6 +35,22 @@ export const AggregatorInfoComponent = ({
     >
       <img alt="exchange icon" src={exchangeIcon} />
     </IconButton>
+    <Box
+      sx={{
+        display: 'grid',
+        gridTemplateColumns: 'repeat(2, 1fr)',
+        gap: '10px 8px',
+        mt: 8,
+        whiteSpace: 'nowrap',
+        color: ({ palette }) => palette.grey[600],
+      }}
+    >
+      <InfoRow
+        label={incentives?.type === IncentiveType.penalty ? 'Penalty' : 'Reward'}
+        value={`${incentives?.amount || '0.0'} XUSD`}
+        state={incentivesState || 'idle'}
```

Comment:
```suggestion
        state={incentivesState}
```
No need for the fallback since `idle` is initial incentive state.